docker/scripts/install_docker.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -ex
+
+DOCKER_VERSION=19.03.8
+
+yum install -y yum-utils device-mapper-persistent-data lvm2
+
+yum-config-manager \
+    --add-repo \
+    https://download.docker.com/linux/centos/docker-ce.repo
+
+# we have to manually install containerd.io
```

Comment:
Is this because a bug somewhere? Do you have a link to any issue tracker?

Reply:
nope, couldn't find anything. it's a conflict, `dnf` wouldn't install the package